src/authors/authors.json

```diff
@@ -4,6 +4,8 @@
     "name": "polubis",
     "firstName": "Adrian",
     "lastName": "Połubiński",
+    "isAuthor": true,
```

Comment:
Instead of flags - you can use "position" - 'author' | 'collaborator' | 'any-other-in-future' - scales much better